linode/instance/resource.go

```diff
@@ -643,6 +643,9 @@ func updateResource(ctx context.Context, d *schema.ResourceData, meta interface{
 		rebootInstance = false
 	}
 
+	// Only reboot the instance if implicit reboots are not skipped
+	rebootInstance = rebootInstance && !meta.(*helper.ProviderMeta).Config.SkipImplicitReboots
```

Comment:
```suggestion
	if !meta.(*helper.ProviderMeta).Config.SkipImplicitReboots {
		rebootInstance = false
	}
```

Just an optional suggestion to align with [the pattern above](https://github.com/linode/terraform-provider-linode/pull/934/files#diff-592f999d8f45773b8b2578d21ecb5d971a279e3ef3903c6a3188202235e92049L642-L644).

Reply:
Unfortunately the new pattern reversed the origin logic: when `skip_implicity_reboots` set as `true` in the config, we should set the `rebootInstance` to be `false`, instead of what we have now. @ykim-1 will put up a fix to remove the `!` in the statement.